modules/upgrade-manual.adoc

```diff
@@ -6,10 +6,10 @@
 :_content-type: PROCEDURE
 [id="upgrade-manual_{context}"]
 
-= Manually upgrading your cluster through {cluster-manager-first}
+= Scheduling individual upgrades for your cluster
 
 
-You can use {cluster-manager} to manually upgrade your {product-title} cluster.
+You can use {cluster-manager} to manually upgrade your {product-title} cluster just once.
```

Comment:
Again, this sounds scary, as if I can set the cluster to update once and never again. 
s/just once/to a specific upgrade (or something similar)?

Reply:
Updated to 'one time'